crates/red_knot_python_semantic/resources/mdtest/directives/assert_type.md

```diff
@@ -78,6 +78,7 @@ def _(a: type[Unknown], b: type[Any]):
 Tuple types with the same elements are the same.
 
 ```py
+from typing import Any
 from typing_extensions import assert_type
```

Comment:
tiny nit: you can import them both from `typing_extensions`, since everything in `typing` is reexported by `typing_extensions`. It makes the setup for the test slightly less verbose:

```suggestion
from typing_extensions import Any, assert_type
```